### PR TITLE
No highlevel fix

### PR DESCRIPTION
**Pull request by anelles**
CoREAS HDF5 files may not have high-level quantities computed. This now allows for the reader to still work but does not compute the weights. Issues a warning.

### CODE REVIEW

File: NuRadioReco/modules/io/coreas/readCoREASStation.py [hunk 1]
```diff
@@ -21,14 +24,20 @@ def begin(self, input_files, station_id):
...
         self.__input_files = input_files
         self.__station_id = station_id
-        self.__current_input_file = 0        
+        self.__current_input_file = 0
 
     def run(self, detector):
         for input_file in self.__input_files:
 
             # read in coreas simulation and copy to event
+#             if input_file == "../../data/CR_sims/005548.hdf5":
```
Comment (christophwelling) [inline NuRadioReco/modules/io/coreas/readCoREASStation.py]: Why do we need this line? We should not hard-code this anyway.

Comment (anelles) [reply NuRadioReco/modules/io/coreas/readCoREASStation.py]: Was commented out already anyway. Removed it for cosmetic reasons now in any case.

**Comment by cg-laser** [comment]
this also means that the coreas reader that randomly assigns core positions already worked with non-highlevel files?

**Comment by cg-laser** [comment]
line 103 in coreas.py also needs attentions

**Comment by anelles** [comment]
No, it just doesn't calculate weights, but no longer crashes. The issue to do this on the fly is still open. I have changed coreas.py in the code cleanup branch.

**Pull request merged**